authenticator/build.gradle.kts

```diff
@@ -295,31 +230,10 @@ protobuf {
     }
 }
 
-sonar {
-    properties {
-        property("sonar.projectKey", "bitwarden_authenticator-android")
-        property("sonar.organization", "bitwarden")
-        property("sonar.host.url", "https://sonarcloud.io")
-        property("sonar.sources", "authenticator/src/")
-        property("sonar.tests", "authenticator/src/")
-        property("sonar.test.inclusions", "authenticator/src/test/")
-        property("sonar.exclusions", "authenticator/src/test/")
-    }
-}
```

Comment:
@vvolkgang if having separate sonar configs for each app is a hard requirement we can add these module-specific settings back. 

I see `projectKey` being set in `test.yml` workflow. I'm guessing this was being overridden anyways by the workflow anyways.